BMM.Core/Translation/en/main.json

```diff
@@ -515,5 +515,14 @@
     "PlaysSingular": "{0} play",
     "AddToMyPlaylist": "Add playlist to favorites",
     "SuccessfullyAddedToPlaylists": "Successfully added to favourites"
+  },
+  "HighlightedTextTrackViewModel": {
+    "AutoTranscribed": "Auto-transcribed",
+    "AddTo": "Add to...",
```

Comment:
It's supposed to be "Add to playlist" and only show ... when there's not enough space.

Reply:
done